src/Model/GroundWaterEnergy/gwe-uze.f90

```diff
@@ -156,10 +159,10 @@ subroutine find_uze_package(this)
     ! -- dummy
     class(GweUzeType) :: this
     ! -- local
-    character(len=LINELENGTH) :: errmsg
     class(BndType), pointer :: packobj
     integer(I4B) :: ip, icount
     integer(I4B) :: nbudterm
+    character(len=LINELENGTH) :: errmsg
```

Comment:
I think the pattern is to import errmsg as `use SimVariablesModule, only: errmsg` at the top so it can be reused throughout the module.

Reply:
Addressed in c25a3a2